sru-scripts/manual/gce-sru

```diff
@@ -44,16 +44,16 @@ ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- systemd-analyze blame
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cloud-init analyze show
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cloud-init analyze blame
 echo 'Networking config'
-ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cat $NETCFG_FILE
+ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cat $NETCFG_FILE || true  # This will fail on xenial
```

Comment:
Hrm NETCFG_FILE should be set on xenial, Ahh it was in https://github.com/cloud-init/ubuntu-sru/pull/127. which just landed. a git fetch upstream; git rebase -i upstream/master should do it and then you don't have to expect failures.

It's worth a change to this script to avoid carrying that confusing comment forward.

Reply:
Ah, gotcha. When I was looking at it it was pointing specifically to a netplan file, which obviously wouldn't exist on xenial. I'll update and rerun for xenial.